wsmaster/che-core-sql-schema/src/main/resources/che-schema/5.20.0/1__add_machine_env.sql

```diff
@@ -0,0 +1,20 @@
+--
+-- Copyright (c) 2012-2017 Red Hat, Inc.
+-- All rights reserved. This program and the accompanying materials
+-- are made available under the terms of the Eclipse Public License v1.0
+-- which accompanies this distribution, and is available at
+-- http://www.eclipse.org/legal/epl-v10.html
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+-- Environment machine env ----------------------------------------------
+CREATE TABLE externalmachine_env (
```

Comment:
I think that script 1 in 5.20.0 already exists

Reply:
It didn't when I created this file. But now it exists. Nice catch. I'll remove it.